tools/snapshot/unittests/export_test.go

```diff
@@ -0,0 +1,96 @@
+package unittests
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"xrplf/clio/clio_snapshot/internal/export"
+	"xrplf/clio/clio_snapshot/mocks"
+	pb "xrplf/clio/clio_snapshot/org/xrpl/rpc/v1"
+
+	"github.com/golang/mock/gomock"
+)
+
+func TestExportDeltaLedgerData(t *testing.T) {
+
```

Comment:
I saw many places with an empty line after function's first line. Is it for some purpose?

Reply:
No , it is not.  I will remove them. Thanks.